### PR TITLE
chore: release fvm 4.2.0

### PR DESCRIPTION
**Pull request by rjan90**
Bump FVM-release to unblock the nv23-skeleton: https://github.com/filecoin-project/filecoin-ffi/pull/452#issuecomment-2063465057

### CODE REVIEW

**Review by Stebalien — APPROVED** [review]

**Comment by rjan90** [comment]
I do not have the right perms to click the merge button it seems. Can you do it for me?

**Comment by Stebalien** [comment]
Ah, actually, now that I look at this more: let's bump to 4.2. We're bumping wasmtime and I believe that's still a slightly breaking change.

**Comment by rjan90** [comment]
Sounds good. Bumped to 4.2.0 now.

File: fvm/CHANGELOG.md [hunk 1]
```diff
@@ -4,6 +4,13 @@ Changes to the reference FVM implementation.
 
 ## [Unreleased]
 
+## 4.2.0 [2023-04-29]
+
+- chore: update to wasmtime 19.0.1 [#1993](https://github.com/filecoin-project/fvm/pull/1993)
+- Enable nv23 support behind the `nv23-dev` feature flag [#2000](https://github.com/filecoin-project/fvm/pull/2000)
+- feat: fvm: remove once_cell [#1989](https://github.com/filecoin-project/fvm/pull/1989)
+- feat: shared: check bls zero address without lazy_static [#1984](https://github.com/filecoin-project/fvm/pull/1984)
```
Comment (rvagg) [inline fvm/CHANGELOG.md:12]: these need fixing, the `fvm` repo redirects to `fvm-website`, I believe these should all be `filecoin-project/ref-fvm`

File: sdk/CHANGELOG.md [hunk 2]
```diff
@@ -2,6 +2,13 @@
 
 ## [Unreleased]
 
+## 4.2.0 [2023-04-29]
+
+- chore: update to wasmtime 19.0.1 [#1993](https://github.com/filecoin-project/fvm/pull/1993)
+- Enable nv23 support behind the `nv23-dev` feature flag [#2000](https://github.com/filecoin-project/fvm/pull/2000)
+- feat: fvm: remove once_cell [#1989](https://github.com/filecoin-project/fvm/pull/1989)
+- feat: shared: check bls zero address without lazy_static [#1984](https://github.com/filecoin-project/fvm/pull/1984)
```
Comment (rvagg) [inline sdk/CHANGELOG.md:10]: ditto

File: shared/CHANGELOG.md [hunk 3]
```diff
@@ -2,6 +2,13 @@
 
 ## [Unreleased]
 
+## 4.2.0 [2023-04-29]
+
+- chore: update to wasmtime 19.0.1 [#1993](https://github.com/filecoin-project/fvm/pull/1993)
+- Enable nv23 support behind the `nv23-dev` feature flag [#2000](https://github.com/filecoin-project/fvm/pull/2000)
+- feat: fvm: remove once_cell [#1989](https://github.com/filecoin-project/fvm/pull/1989)
+- feat: shared: check bls zero address without lazy_static [#1984](https://github.com/filecoin-project/fvm/pull/1984)
```
Comment (rvagg) [inline shared/CHANGELOG.md:10]: ditto

File: fvm/CHANGELOG.md (hunk 1 above)
Comment (rjan90) [reply fvm/CHANGELOG.md]: Thanks for catching. Fixed in: https://github.com/filecoin-project/ref-fvm/pull/2002/commits/a50538eedef656996048af2b8e5621acde7777c9

**Review by rvagg — APPROVED** [review]

**Pull request merged**